File: psets/pset6/cash/cash.py
from cs50 import get_float

dollars = 0
coins = 0

while dollars <= 0:
    dollars = get_float("Change owed: ")

cents = round(dollars * 100)

if cents >= 25:
    coins += cents // 25
    # print(coins)
    cents %= 25

if cents >= 10:
    coins += cents // 10
    # print(coins)
    cents %= 10

if cents >= 5:
    coins += cents // 5
    # print(coins)
    cents %= 5

if cents > 0:
    coins += cents
    # print(coins)


print(coins)

File: psets/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    // getting the input
    float dollar = 0;
    do
    {
        dollar = get_float("Change owed: ");
    }
    while (dollar <= 0);

    // finding the cents from the given user input
    int cents = round(dollar * 100);

    // intializing variable to store the count of coins
    int count = 0;

    // no of 25c coins
    if (cents >= 25)
    {
        count += cents / 25;
        cents %= 25;
    }

    // no of 10c coins
    if (cents >= 10)
    {
        count += cents / 10;
        cents %= 10;
    }

    // no of 5c coins
    if (cents >= 5)
    {
        count += cents / 5;
        cents %= 5;
    }

    // no of 1c coins
    if (cents >= 1)
    {
        count += cents;
    }

    // display the count to the user
    printf("%i\n", count);
}